### ISSUE TITLE
Assert statements have no effect

### ISSUE THREAD

**Issue by zenware** [rationalmatter/Juno-Issue-Tracker#114]
**Describe the bug**
If you have a failing assert statement in a cell and run that cell, it doesn’t show an AssertionError

**To Reproduce**
1. Type ‘assert 1 == 2` into a cell
2. Run the cell
3. See nothing at all

**Expected behavior**
I should se an AssertionError

**Screenshots**
N/A, the issue is absence of info

**Environment (please complete the following information):**
 - Device: iPad Pro 11" 256GB WiFi + Cellular
 - iOS version: iOS 13.3 (17C54)
 - App version: 1.1.1 (4128)

**Comment by zenware** [comment]
Also, happy holidays everyone!

**Comment by zenware** [comment]
A more visual way to repro is
```
try:
    assert False
    raise Exception('Python Assertions are not working. This tool relies on Python Assertions to do its job. Possible causes are running with the "-O" flag or running a precompiled (".pyo" or ".pyc") module.')
except AssertionError:
    pass
```
Or perhaps `print(__debug__)`

**Comment by navoshta** [comment]
Thanks for reporting @zenware, and apologies for a late reply! Yes, you are right: interpreter is running with the `-O` flag (or rather `PYTHONOPTIMIZE` set to true) — I didn't realise this would also ignore assertions in notebooks (duh!). I'll remove this in the next release (after I make sure this doesn't introduce any side effects).

**Comment by zenware** [comment]
I think there must be side-effects, though I typically expect my playbooks to run in debug mode rather than “production” mode. 😁

Thanks for looking into this for me.

**Comment by navoshta** [comment]
@zenware Juno 1.2 is now waiting for TestFlight review, and should be available in the next few days — please send an e-mail to beta@juno.sh if you would like to help test it! New version should now allow assert statements.

**Comment by navoshta** [comment]
This should be fixed in Juno 1.2 (now live on App Store)
